composer.json

```
@@ -14,9 +14,9 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
-        "phpunit/phpunit": "^8.0 || ^9.3",
-        "psr/http-message": "^1.0 || ^2.0"
+        "php": "^7.3 || ^8.0",
+        "phpunit/phpunit": "^9.3 || ^10.0",
+        "psr/http-message": "^1.0"
```

```suggestion
        "psr/http-message": "^1.0 || ^2.0"
```

we should keep allowing http-message 2.0. i think this was one of the merge conflicts.